plugin/health/README.md

```diff
@@ -33,8 +33,9 @@ health [ADDRESS] {
 * Where `lameduck` will make the process unhealthy then *wait* for **DURATION** before the process
   shuts down.
 
-If you have multiple Server Block and need to export health for each of the plugins, you must run
-health endpoints on different ports:
+If you have multiple Server Block and need to export health for each of the plugins, 
+you can either run all health enpoints on the same port, or on different ports. 
+In that case, 
```

Comment:
Not clear what case you are referring too, the one with same ports or different ports.

Also not completely clear on what the second Corefile shows here. (Or the third)